pico/src/scbus.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2024 Space Cubics, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <stdint.h>
```

Comment:
You don't need this in the header.  you need it in `.c` because you use `uint32_t`.